baseplate/context/memcache/__init__.py

```diff
@@ -5,11 +5,12 @@
 
 from pymemcache.client.base import PooledClient
 
-from . import ContextFactory
-from .. import config
+from .lib import make_memcache_serializer, memcache_deserializer
+from baseplate.context import ContextFactory
+from baseplate import config
 
 
-def pool_from_config(app_config, prefix="memcache.", **kwargs):
```

Comment:
we make use of kwargs to set serde right now, maybe keeping it in and letting services override this serde would be nice: https://github.com/reddit/reddit-service-listing/blob/a9668dea8ee28e5e1dbb9fbdf7ca52a77e33c994/listing/__init__.py#L113